chatcommunicate.py

```diff
@@ -59,7 +58,10 @@ def special_room_watcher(ev, wrap2):
         post_site_id = fetch_post_id_and_site_from_msg_content(content_source)
         post_url = fetch_post_url_from_msg_content(content_source)
         if post_site_id is not None and post_url is not None:
-            t_check_websocket = Thread(target=DeletionWatcher.check_if_report_was_deleted, args=(post_site_id, post_url, ev.message))
```

Comment:
Also, do we really need to wrap everything to 80 characters? I believe that was something we specifically didn't want to do in the beginning -- 80 characters is a large restriction (https://github.com/Charcoal-SE/SmokeDetector/commit/127d2f743cbe35ec00768b00d1b5b2ffa20cb68b#diff-b91f3d5bd63fcd17221b267e851608e8R2)